src/main/java/com/example/humorie/admin/dto/NoticeDto.java

```diff
@@ -0,0 +1,31 @@
+package com.example.humorie.admin.dto;
+
+import com.example.humorie.notice.entity.Notice;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class NoticeDto {
+    private Long id;
+    private String title;         // 제목
+    private String content;       // 내용
+    private int viewCount;        // 조회수
+    private String adminName;     // 관리자 이름
+    private LocalDate createdDate; // 생성 날짜
+    private LocalTime createdTime; // 생성 시간
```

Comment:
```suggestion
    private Long id;
    @NotBlank(message = "제목을 입력해주세요.")
    @Max(value = 50, message = "제목은 최대 50자까지 작성 가능합니다.")
    private String title;         // 제목
    @NotBlank(message = "내용을 입력해주세요")
    @Max(value = 1000, message = "내용은 최대 1000자까지 작성 가능합니다.")
    private String content;       // 내용
    private int viewCount;        // 조회수
    private String adminName;     // 관리자 이름
    private LocalDate createdDate; // 생성 날짜
    private LocalTime createdTime; // 생성 시간
```

이런 식으로 작성하면 Controller 단에서 @Valid 어노테이션으로 검증이 끝난다고 하네요! 참고 해주시면 좋을 것 같습니다!